packages/react-native/ReactCommon/react/nativemodule/core/platform/ios/ReactCommon/RCTTurboModule.mm

```diff
@@ -211,6 +211,16 @@ static int32_t getUniqueId()
   return [callback copy];
 }
 
+static NSString *convertNSExceptionStackToString(NSException *exception)
+{
+  NSMutableString *stack = [[NSMutableString alloc] init];
+  for (NSString* symbol in exception.callStackSymbols) {
```

Comment:
This will not be symbolicated in most builds. Can you use a format similar to the standard objc exception printer that includes both returnAddress and symbols?

Reply:
Yes, this will work mostly for debug builds, but in release, it would not be helpful.

Definitely, I will add the return addresses.